### PR TITLE
(refactor) Better type annotations

### PR DESCRIPTION
**Pull request by denniskigen**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [ ]  My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [x] My work includes tests or is validated by existing tests.

This PR applies a bunch of refactors and adds type annotations to existing code.

### CODE REVIEW

File: e2e/commands/formOperations.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { APIRequestContext } from "@playwright/test";
 
-export const deleteForm = async (api: APIRequestContext, uuid: string) => {
+export async function deleteForm(api: APIRequestContext, uuid: string) {
```
Comment (anjula-sack) [inline e2e/commands/formOperations.ts:3]: @denniskigen Why can't we use arrow functions here?

Comment (denniskigen) [reply e2e/commands/formOperations.ts:3]: No particular reason (except personal preference, maybe IIRC). Happy to revert if the former is more readable.

Comment (denniskigen) [reply e2e/commands/formOperations.ts:3]: Does everything else look ok, @anjula-sack?

**Review by anjula-sack — APPROVED** [review]
LGTM!

File: e2e/commands/formOperations.ts (hunk 1 above)
Comment (anjula-sack) [reply e2e/commands/formOperations.ts:3]: yes

**Pull request merged**